iron-resizable-behavior.html

```diff
@@ -61,6 +61,20 @@
     },
 
     attached: function() {
+      // NOTE(valdrin) In CustomElements v1 with native HTMLImports, the order
+      // of imports affects the order of `attached` callbacks (see webcomponents/custom-elements#15).
+      // This might cause a child to notify parents too early (as the parent
+      // still has to be upgraded), resulting in a parent not able to keep track
+      // of the `_interestedResizables`. To solve this, we wait for the document
+      // to be done loading before firing the event.
+      if (document.readyState === 'loading') {
+        var _this = this;
+        document.addEventListener('readystatechange', function readystatechanged() {
+          document.removeEventListener('readystatechange', readystatechanged);
+          _this.isAttached && _this.attached();
+        });
+        return;
+      }
```

Comment:
For the general case of elements in the main document that get imported backwards (and don't have the pathological nesting mentioned above), waiting for `document.readyState !== 'loading'` should be sufficient.